packages/manager/apps/telecom/src/app/telecom/telephony/service/voicemail/default/telecom-telephony-service-voicemail-default.html

```diff
@@ -19,16 +19,16 @@
                         data-translate="telephony_line_answer_default_voicemail_label"
                     >
                     </label>
-                    <select
+                    <input
```

Comment:
Do you think a help text like "enter first few numbers to search" will help users?

Reply:
You're right, a placeholder could be nice, I'll add it